codespell_lib/data/dictionary.txt

```diff
@@ -4587,6 +4589,7 @@ combanations->combinations
 combatibility->compatibility
 combatible->compatible
 combiantion->combination
+combiations->combinations
```

Comment:
Please add the equivalent singular typo and correction.

Reply:
Done